sys-auth/AusweisApp2/AusweisApp2-9999.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake git-r3
+
+DESCRIPTION="Official authentication app for German ID cards and residence permits"
+HOMEPAGE="https://www.ausweisapp.bund.de/"
+EGIT_REPO_URI="https://github.com/Governikus/AusweisApp2.git"
+
+LICENSE="EUPL-1.2"
+SLOT="0"
+KEYWORDS=""
+IUSE=""
+
+BDEPEND="virtual/pkgconfig"
+
+RDEPEND="
+	dev-qt/qtcore:5
+	dev-qt/qtconcurrent:5
```

Comment:
I don't see this depdency used? It that linked with your local build?

Reply:
sometimes there is header only usage of this package, needs to be grep'd.

Reply:
I will look later in the source code. Haven't yet digged deep into it.

Reply:
I am just re-testing those dependencies.

Reply:
-- BUILD_SHARED_LIBS: OFF
CMake Error at cmake/Libraries.cmake:19 (FIND_PACKAGE):
  By not providing "FindQt5QuickControls2.cmake" in CMAKE_MODULE_PATH this
  project has asked CMake to find a package configuration file provided by
  "Qt5QuickControls2", but CMake did not find one.

  Could not find a package configuration file provided by "Qt5QuickControls2"
  (requested version 5.10) with any of the following names:

    Qt5QuickControls2Config.cmake
    qt5quickcontrols2-config.cmake

Reply:
-- CMAKE_SYSROOT_COMPILE: 
-- CMAKE_SYSTEM_PROCESSOR: x86_64
-- CMAKE_CXX_SIMULATE_ID: 
-- BUILD_SHARED_LIBS: OFF
CMake Error at cmake/Libraries.cmake:10 (FIND_PACKAGE):
  By not providing "FindQt5Concurrent.cmake" in CMAKE_MODULE_PATH this
  project has asked CMake to find a package configuration file provided by
  "Qt5Concurrent", but CMake did not find one.

  Could not find a package configuration file provided by "Qt5Concurrent"
  (requested version 5.10) with any of the following names:

    Qt5ConcurrentConfig.cmake
    qt5concurrent-config.cmake

  Add the installation prefix of "Qt5Concurrent" to CMAKE_PREFIX_PATH or set
  "Qt5Concurrent_DIR" to a directory containing one of the above files.  If
  "Qt5Concurrent" provides a separate development package or SDK, be sure it
  has been installed.
Call Stack (most recent call first):
  CMakeLists.txt:145 (INCLUDE)

Reply:
Now moved to DEPEND